PowervRA/Functions/Public/config-service/Disconnect-vRAVAMI.psm1

```diff
@@ -0,0 +1,48 @@
+function Disconnect-vRAVAMIServer {
+<#
+    .SYNOPSIS
+    Disconnect from a vRA appliance
+   
+    .DESCRIPTION
+    Disconnect from a vRA appliance by removing the authorization token and the global vRAConnection variable from PowerShell
+ 
+    .EXAMPLE
+    Disconnect-vRAVAMIServer
+   
+    .EXAMPLE
+    Disconnect-vRAVAMIServer -Confirm:$false
+#>
+[CmdletBinding(SupportsShouldProcess,ConfirmImpact="High")]
+ 
+    Param ()
+ 
+    # --- Test for existing connection to vRA
+    if (-not $Global:vRAVAMIConnection){
+ 
+        throw "vRA Connection variable does not exist. Please run Connect-vRAServer first to create it"
+    }
```

Comment:
Change to "vRAVAMI Connection variable........Please run Connect-vRAVAMI first...."